### PR TITLE
Update cmd of read_rng_cmd

### PR DESCRIPTION
**Pull request by suqinhuang**
Signed-off-by: Suqin Huang <shuang@redhat.com>

id: 1377564

### CODE REVIEW

**Comment by suqinhuang** [comment]
PASS 1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.rng_read_longtime

**Comment by CongLi** [comment]
@lijinlijin @huangyum 

Could you help review this patch?

Thanks.

File: qemu/tests/cfg/rng_read_longtime.cfg [hunk 1]
```diff
@@ -21,7 +21,7 @@
             driver_check_cmd = "WIN_UTILS:\devcon\wnet_amd64\devcon.exe status @DRIVER_ID"
     Linux:
         session_cmd_timeout = 360
-        read_rng_cmd = "dd if=/dev/random bs=4 count=100 2>/dev/null|hexdump"
+        read_rng_cmd = "dd if=/dev/random bs=4 count=10000 2>/dev/null|hexdump"
```
Comment (huangyum) [inline qemu/tests/cfg/rng_read_longtime.cfg]: What's the reason to change `count` to 10000 please?

Comment (suqinhuang) [reply qemu/tests/cfg/rng_read_longtime.cfg]: 100 is used before as it's slow in amd host, now /dev/urandom is update to /dev/random, 10000 is  acceptabled in amd host.

rng read is slow in amd host quick in intel host before, which cause the test failed. 10000 is ok for both now

**Comment by huangyum** [comment]
(1/1) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.rng_read_longtime: PASS (37000.12 s)

Acked-by: Yumei Huang <yuhuang@redhat.com>

**Comment by huangyum** [comment]
Test with win2016 guest failed.
 (1/1) smp_16.8192m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.rng_read_longtime: ERROR (722.17 s)

01:30:46 ERROR| Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado_plugins_vt-54.0-py2.7.egg/avocado_vt/test.py:245
01:30:46 ERROR| Traceback (most recent call last):
01:30:46 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-54.0-py2.7.egg/avocado_vt/test.py", line 397, in _runTest
01:30:46 ERROR|     run_func(self, params, env)
01:30:46 ERROR|   File "/usr/lib/python2.7/site-packages/avocado_plugins_vt-54.0-py2.7.egg/virttest/error_context.py", line 135, in new_fn
01:30:46 ERROR|     return fn(*args, **kwargs)
01:30:46 ERROR|   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-qemu/qemu/tests/rng_bat.py", line 127, in run
01:30:46 ERROR|     timeout=read_rng_timeout)
01:30:46 ERROR|   File "/usr/lib/python2.7/site-packages/aexpect/client.py", line 1086, in cmd_output
01:30:46 ERROR|     raise ShellTimeoutError(cmd, o)
01:30:46 ERROR| ShellTimeoutError: Timeout expired while waiting for shell command to complete: u'for /l %i in (1, 1, 10000) do "D:\\random_%PROCESSOR_ARCHITECTURE%.exe"'    (output: u'\nC:\\>"D:\\random_AMD64.exe"\nGenerated random number is\n0x16, 0xd2,...

**Comment by suqinhuang** [comment]
amd:
PASS 1-smp_2.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.rng_read_longtime

PASS 2-smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.rng_read_longtime

intel:
PASS 1-smp_2.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.rng_read_longtime

PASS 2-smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.rng_read_longtime

**Comment by suqinhuang** [comment]
@huangyum updated

**Comment by lijinlijin** [comment]
Looks good to me.
Acked-by: lijin@redhat.com

**Comment by CongLi** [comment]
Ack.

**Comment by huangyum** [comment]
Test with win2016 guest for 1 hour, and no error appears.

Acked-by: Yumei Huang yuhuang@redhat.com

**Comment by CongLi** [comment]
Thanks all.

**Pull request merged**